### ISSUE TITLE
incoterm_id defined in purchase AND purchase_stock

### ISSUE THREAD

**Issue by alexis-via** [odoo/odoo#65347]
On odoo v13, v14 and master, the field incoterm_id on purchase.order is defined both in the module purchase (https://github.com/odoo/odoo/blob/staging.master/addons/purchase/models/purchase.py#L123) and purchase_stock (https://github.com/odoo/odoo/blob/staging.master/addons/purchase_stock/models/purchase.py#L19)

It works, but this is kind of "dirty". It should be fixed in master.

**Comment by pedrobaeza** [comment]
What about doing a PR for master removing one of them?

**Comment by alexis-via** [comment]
Yes, we just need to decide in which module the field should be defined. On the sale side, that field is defined in sale_stock, so I suggest to define the field in purchase_stock.

**Comment by pedrobaeza** [comment]
Yeah, I agree. It's something linked to stock.

**Comment by Yenthe666** [comment]
Yep, sounds like the best idea :) Feel free to make a PR.
